ci/run_fuzzit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+git clone https://github.com/google/oss-fuzz.git
+export ENVOY_SRC_PATH=`pwd`
+cd oss-fuzz
+python infra/helper.py build_image --pull envoy
```

Comment:
Are we doing nested Docker here? I.e. running oss-fuzz Docker build inside a CircleCI Docker env?

Reply:
I'm exactly working on this to make it work not inside the oss-fuzz docker.